Src/Base/AMReX_MultiFabUtil.H

```diff
@@ -329,6 +329,137 @@ void average_down_nodal (const FabArray<FAB>& fine, FabArray<FAB>& crse,
     }
 }
 
+   /**
+    * \brief Returns part of a norm based on two MultiFabs
+    * The MultiFabs MUST have the same underlying BoxArray.
+    * The Predicate pf is used to test the mask
```

Comment:
This one has no predicate pf.  But we can probably remove this version because we should be able to call the other one with a pf that is known to the compiler to return true.

Reply:
Never mind my comment.  The predicate requires a mask multifab.  But the comment still needs to be fixed.